### DISCUSSION TITLE
Transfer IT domain

### DISCUSSION THREAD

**Discussion by nicolocurioni96** [vercel/community#3034] [Help] [answered]
Hi everyone, it's a pleasure to be here with u!

I need help on a domain, basically, an .it domain.

I have a domain, for a project that I'm working on, but the only domain I found (outside Vercel domain) are a different provider but, with .it.

Because for .io or .com, are not available, but, I have found a .it, but I got a trouble.

Can someone at Vercel, help me to transfer (with the transfer code), a domain here?

I hope someone could help me.

Kind regards,

Nicolò 🟠

**Comment by amyegan** [comment]
Hi @nicolocurioni96! We have a guide to walk you through domain transfers: https://vercel.com/guides/how-do-i-transfer-my-domain-to-vercel

If you prefer to keep your domain with the other registrar, you can still use the domain with your project by following these instructions: https://vercel.com/docs/concepts/projects/domains/add-a-domain

**Reply by nicolocurioni96** [reply]
Hi @amyegan, thanks for your response!

Yes, I know, but I tried this, when transferring my domain, and I got this (in screenshot):

![image](https://github.com/vercel/community/assets/102799475/53dce3a4-a593-47f7-88f1-df1ab6615c0c)

How can I do ?

Thanks,

Nicolò

**Reply by smaeda-ks** [reply, marked as answer]
@nicolocurioni96 We currently cannot support transferring in `.it` TLD domains, so consider using other methods, such as switching your name servers to Vercel ([docs](https://vercel.com/docs/concepts/projects/domains/troubleshooting#configuring-nameservers-for-wildcard-domains)), or just using your current domain registrar. Transferring-in domains isn't a hard requirement to start using Vercel.
